triaging/tracker.rst

```diff
@@ -1,6 +1,6 @@
-==============
-Issue Tracking
-==============
+===============
+ Issue Tracker
+===============
```

Comment:
.. _tracker:
.. _issue-tracker:
.. _issues:

```suggestion
=============
Issue Tracker
=============
```

Move this up here where it belongs, and add aliases used elsewhere for links